api/client/commands.go

```diff
@@ -2735,7 +2735,7 @@ func (cli *DockerCli) CmdStats(args ...string) error {
 	if len(errs) > 0 {
 		return fmt.Errorf("%s", strings.Join(errs, ", "))
 	}
-	for _ = range time.Tick(500 * time.Millisecond) {
+	for range time.Tick(500 * time.Millisecond) {
```

Comment:
Don't we need to keep the `for _ = range time.Tick` syntax here for Go 1.3 support?